diff --git a/packages/checkseum/checkseum.0.3.2/opam b/packages/checkseum/checkseum.0.3.2/opam
index 3a50559f7c..176dd781b9 100644
--- a/packages/checkseum/checkseum.0.3.2/opam
+++ b/packages/checkseum/checkseum.0.3.2/opam
@@ -17,7 +17,7 @@ top of optint to get the best representation of an int32. """
 
 build: [
   [ "dune" "build" "-p" name "-j" jobs ]
-  [ "./install/install.ml" ]
+  [ "%{ocaml:bin}%/ocaml" "./install/install.ml" ]
   [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
 ]
 
