diff --git a/packages/digestif/digestif.1.1.0/opam b/packages/digestif/digestif.1.1.0/opam
index aaec4bb64f..98c874576e 100644
--- a/packages/digestif/digestif.1.1.0/opam
+++ b/packages/digestif/digestif.1.1.0/opam
@@ -28,12 +28,12 @@ We provides implementation of:
 
 build: [
   [ "dune" "build" "-p" name "-j" jobs ]
-  [ "./install/install.ml" ]
+  [ "%{ocaml:bin}%/ocaml" "./install/install.ml" ]
   [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
 ]
 install:  [
   [ "dune" "install" "-p" name ] {with-test}
-  [ "./test/test_runes.ml" ] {with-test}
+  [ "%{ocaml:bin}%/ocaml" "./test/test_runes.ml" ] {with-test}
 ]
 
 depends: [
